packages/flutter/lib/src/rendering/viewport.dart

```diff
@@ -171,17 +171,21 @@ abstract class RenderViewportBase<ParentDataClass extends ContainerParentDataMix
     @required ViewportOffset offset,
     double cacheExtent,
     CacheExtentStyle cacheExtentStyle = CacheExtentStyle.pixel,
+    Clip clipBehavior = Clip.hardEdge,
```

Comment:
ClippableRenderBox claims that the default is none.

Reply:
I don't think dartdoc provides any affordances for this, but I am no dartdocs expert.

But the auto-generated documentation for all classes that extend ClippableRenderBox or mixin the mixin and set the default to something other then "none" will have incorrect documentation with the current approach.

Reply:
I wouldn't bother with the mixin. Just inline the duplicate code. It's not complex logic, and it might not be identical everywhere (as shown by the different defaults, for one).

Reply:
Done.